src/send_packets.c

```diff
@@ -93,11 +94,37 @@ float2timer (float time, struct timeval *tvp)
     tvp->tv_usec = n * 100000;
 }
 
-/* buffer should be "file_name" */
-int
-parse_play_args (char *buffer, pcap_pkts *pkts)
+static char* find_file(const char* filename)
 {
-    pkts->file = strdup (buffer);
+    if (filename[0] == '/' || !scenario_path) {
+        return strdup(filename);
+    }
+
+    char* fullpath = NULL;
+    if (asprintf(&fullpath, "%s/%s", scenario_path, filename) < 0) {
```

Comment:
Yeah, I don't consider this a final version. Technically we're already relying on other GNU extensions, so I thought it be okay for a review. Two things I wanted to bring up.

- `MAX_PATH` is broken everywhere but Windows. And SIPp hardcodes an even smaller maximum that the platforms it supports (ie: for linux its 4098, the code hardcodes 512).
- `asprintf` is useful and easy to implement for platforms that don't have it (call `snprintf` twice).

But yeah, I can just use `MAX_PATH` here like everywhere else in the code base and tackle that issue all at once separately.